### DISCUSSION TITLE
OpuntiaOS / BrutalOS

### DISCUSSION THREAD

**Discussion by TheArchitectEngineer** [opuntiaOS-Project/opuntiaOS#56] [General]
Hi

Take a look at this project, I wonder if the project below is your code

https://github.com/evilbat831/brutalOS

Spark

**Comment by nimelehin** [comment]
Hey! Thanks for the report. Seems like it is a fork of [pranaOS which had this project's codebase](https://github.com/pranaOS/pranaOS/tree/8e35bd5c9db2401cfbf9553ed062d6f145b0101f) for some time.
